dev_contain/run.py

```diff
@@ -48,15 +48,17 @@ def run(in_args):
     if os.path.exists('/home/{}/.ssh'.format(username)):
         ssh_text = '--volume /home/{}/.ssh:/home/{}/.ssh:Z'.format(username, username)
 
-    command = ('podman run --rm'
+    # Attach with 
+    # podman exec -i -t dev tmux new -A -s dev
```

Comment:
Add a python function that will do this.
Also need way to clean up.

Can we assume always will have tmux for dev_contain containers?